airflow/gcp/hooks/gcs.py

```diff
@@ -336,13 +355,11 @@ def list(self, bucket_name, versions=None, max_results=None, prefix=None, delimi
     def get_size(self, bucket_name, object_name):
         """
         Gets the size of a file in Google Cloud Storage.
-
```

Comment:
This change prevents the documentation from being built properly. Can you delete it?